app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,65 @@
+
+<div class="border border-3 border-primary container rounded recipe-c">
+  <div class="my-3">
+    <h2><%= @recipe.name%></h2>
+  </div>
+  <div>
+    <div class="split">
+      <div class="d-flex justify-content-between p-2">
+        <h5> Preparation time: <%= @recipe.preparationTime %> min </h5>
+        <div class="public-btn">
+        <% if @recipe.public %>
+          <div class="form-check form-switch">
+            <input class="form-check-input" type="checkbox" id="flexSwitchCheckChecked" checked>
+            <label class="form-check-label" for="flexSwitchCheckChecked">Public</label>
+          </div>
+        <% end %>
+        <% if !@recipe.public %>
+          <div class="form-check form-switch">
+            <input class="form-check-input" type="checkbox" id="flexSwitchCheckDefault">
+            <label class="form-check-label" for="flexSwitchCheckDefault"></label>
+          </div>
+        <% end %>
+      </div>
+      </div>
+    </div>
+    <div>
+      <h5 class="p-2"> Cooking time: <%= @recipe.cookingTime %> min </h5>
+      <h5 class="p-2"> Steps go here...</h5>
+    </div>
+    <div class="d-flex justify-content-around">
+     <% if user_signed_in? %>
+    <div class="recipe-btn">
+      <%= link_to "Generate Shopping list",  recipe_shopping_lists_path(@recipe.id), :class => "btn btn-primary btn-gen" %>
+      <%= link_to "Add ingredient", new_recipe_recipe_food_path(@recipe) , class: "btn btn-primary btn-gen "%>
+    </div>
+    <% end %>
+    </div>
+  </div>
+</div>
+<table class="table table-bordered table-striped recipe-table ">
+  <thead>
+    <tr>
+      <th scope="col">Food</th>
+      <th scope="col">Quantity</th>
+      <th scope="col">Value</th>
+      <th scope="col">Action</th>
+    </tr>
+  </thead>
+  <tbody>
+  <% @recipe.recipe_foods.each do |food| %>
+    <tr>
+      <th scope="row"><%= food.food.name %></th>
+      <td><%= food.quantity %></td>
+      <td><%= food.food.Price * food.quantity %></td>
```

Comment:
- [ ] Kindly add the measuring unit to the food quantity and also add currency value e.g `$` to the calculated value.

Your Implementation | Wireframe
:---:|:---:
![](https://i.imgur.com/OkKC9uE.png) | ![](https://i.imgur.com/O2rAncG.png)

```suggestion
      <td><%= food.quantity %><%= food.food.measurementUnit %></td>
      <td>$<%= food.food.Price * food.quantity %></td>
```

Reply:
Thanks for your feedback. We fixed the issue.